src/sonic-host-services/setup.py

```diff
@@ -23,13 +23,14 @@
     ],
     setup_requires = [
         'pytest-runner',
-        'wheel'
+        'wheel',
```

Comment:
>, [](http://example.com/codeflow?start=15&length=1)

This is not in old PR. I am always wondering how do you cherry-pick and resolve conflicts locally.

Reply:
I'm sorry. Removed this comma. Cherry pick in this case wasn't possible. As the code for Feature Handler has diverged a lot that i thought it'll be easy to rewrite instead